recipes/bcolz/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "bcolz" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "42b733692cfb194df867f19168da1a9e468145a50c557758ecec911c6e5a4073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - setuptools_scm
+    - cython
+    - numpy x.x
+  run:
+    - python
+    - numpy x.x
+    - numexpr
+    - dask
+    - pandas
+    - pytables
+
+test:
+  requires:
+    - mock
+  imports:
+    - bcolz
+  commands:
+    - python -c"import bcolz; bcolz.test(heavy=True)"
+    - python -c"import bcolz; assert set(bcolz.blosc_compressor_list()) == set(['blosclz', 'lz4', 'lz4hc', 'snappy', 'zlib', 'zstd'])"
```

Comment:
For this sort of stuff, it may be better to move it to `run_test.py`. Should make it easier to read and follow. Plus it just runs with `python`. So there is no need for the `python -c "..."`.